src/resolvers/hosts/hosts.integration.ts

```diff
@@ -1105,6 +1131,56 @@ describe('hosts query', function () {
                 expect(data).toMatchSnapshot();
             });
         });
+
+        describe('per_reporter_staleness', function () {
+            test('reporter', async () => {
+                const { data } = await runQuery(PRS_QUERY,
+                    { filter: { per_reporter_staleness: { reporter: { eq: 'yupana' }}}});
+                data.hosts.data.should.have.length(2);
+                data.hosts.data[0].id.should.equal('6e7b6317-0a2d-4552-a2f2-b7da0aece49d');
```

Comment:
Only the system with id `f5ac67e1-ad65-4b62-bc27-845cc6d4bcee` has reporter=yupana. That's why this test is failing.

Reply:
Oops - I didn't realize that my recent fix broke something else. I'll get that fixed ASAP